keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -299,6 +299,9 @@ the API proposed by this KEP aims to
   clusters under management.
 
 ### Terminology
+- **Cluster Inventory**: A collection of clusters governed by a single authority. These clusters SHOULD
```

Comment:
1. Should we clarify that the ClusterInventory is the List of ClusterProfiles in the same namespace and that ClusterProfile is namespaced for that reason? 
2. I'd say "MUST adhere to sameness"; because it could break controller assumptions.
3. I'd skip the definition of sameness in favor of just the link (because I disagree with "generally behave the same", it's a "same identity"... they are the same thing as far as other services care)
4. You don't mention ClusterSet. It sounds like we could replace sameness by clusterset since its already defined? Or are we giving up on ClusterSet? And instead re-assigning sameness to be at the "Namespace of ClusterProfiles"?